api/v1/clusterextensionrevision_types.go

```diff
@@ -25,21 +25,16 @@ const (
 	ClusterExtensionRevisionKind = "ClusterExtensionRevision"
 
 	// Condition Types
-	ClusterExtensionRevisionTypeAvailable = "Available"
-	ClusterExtensionRevisionTypeSucceeded = "Succeeded"
+	ClusterExtensionRevisionTypeAvailable   = "Available"
+	ClusterExtensionRevisionTypeSucceeded   = "Succeeded"
+	ClusterExtensionRevisionTypeProgressing = "Progressing"
 
 	// Condition Reasons
-	ClusterExtensionRevisionReasonAvailable                 = "Available"
-	ClusterExtensionRevisionReasonReconcileFailure          = "ReconcileFailure"
-	ClusterExtensionRevisionReasonRevisionValidationFailure = "RevisionValidationFailure"
-	ClusterExtensionRevisionReasonPhaseValidationError      = "PhaseValidationError"
-	ClusterExtensionRevisionReasonObjectCollisions          = "ObjectCollisions"
-	ClusterExtensionRevisionReasonRolloutSuccess            = "RolloutSuccess"
-	ClusterExtensionRevisionReasonProbeFailure              = "ProbeFailure"
-	ClusterExtensionRevisionReasonIncomplete                = "Incomplete"
-	ClusterExtensionRevisionReasonProgressing               = "Progressing"
-	ClusterExtensionRevisionReasonArchived                  = "Archived"
-	ClusterExtensionRevisionReasonMigrated                  = "Migrated"
+	ClusterExtensionRevisionReasonRetrying   = "Retrying"
+	ClusterExtensionRevisionReasonBlocked    = "Blocked"
+	ClusterExtensionRevisionReasonRollingOut = "RollingOut"
+
+	ClusterExtensionRevisionReasonArchived = "Archived"
```

Comment:
Does it make sense to sort these alphabetically

Reply:
done